pyscriptjs/src/main.ts

```diff
@@ -196,35 +186,66 @@ export class PyScriptApp {
         }
 
         const interpreter_cfg = this.config.interpreters[0];
+        const useWorker = this.config.execution_thread == 'worker';
+
+        if (useWorker) {
+            logger.info('Starting the interpreter in a web worker');
+            // XXX what is the best way to specify a robust URL? I want to
+            // load a sibling of the current file (i.e. pyscript.js), but I
+            // don't know how to specify that.
```

Comment:
See this comment:
https://github.com/pyodide/pyodide/blob/main/src/js/pyodide.ts#L123-L140
you can probably use `document.currentScript`.

Reply:
thanks! Fixed by 0eee2dc